### PR TITLE
AT_01.02.028 | <Header> Verify Search box is case insensitive by default

### PR DESCRIPTION
**Pull request by milladiz84**
https://trello.com/c/IUccaecI/1197-at0102028-header-verify-search-box-is-case-insensitive-by-default

### CODE REVIEW

File: cypress/e2e/headerSearchBox.cy.js [hunk 1]
```diff
@@ -110,4 +110,12 @@ describe('Header Search Box', () => {
   it('AT_01.02_023 | Validation of the Search box', ()=> { 
     cy.get('#search-box').should('have.attr','placeholder','Search (CTRL+K)') 
   });
+
+  it('AT_01.02.028 | Verify Search box is case insensitive by default', () => {
+    headers.dataSearchBox.forEach(arr => {      
+        cy.get('input#search-box').clear().type(arr + '{enter}');
+        cy.get('a[href="all"]').should('have.text', 'All Jenkins Logs');
```
Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:117]: Please move `'All Jenkins Logs'` in the fixtures

Comment (milladiz84) [reply cypress/e2e/headerSearchBox.cy.js]: Fixed it. Thanks.

**Review by Rom-T — APPROVED** [review]

**Review by OLgaByR — APPROVED** [review]

File: cypress/e2e/headerSearchBox.cy.js (hunk 1 above)
Comment (OLgaByR) [reply cypress/e2e/headerSearchBox.cy.js]: please be advised one PR should contain only one it)

Comment (milladiz84) [reply cypress/e2e/headerSearchBox.cy.js]: > please be advised one PR should contain only one it)

I have one test case and one IT for it. Above me are written IT by other members.

**Pull request merged**